SU-HSDO/suhumsci
# fix(STN-873) lockup pipe bug on chrome zoom

# READY FOR REVIEW

## Summary
Zooming out at 67% view on Chrome shows double pipe in lockup.

JIRA Card: https://sparkbox.atlassian.net/browse/STN-873

## Work included in this card
In the asset injector, the right border on Cell1 of the lockup was removed. This work makes that fix permanent.
There were some small concerns with the Cell2 border being the border to control the border completely as the sizes of each cell could be different, therefore, the borders being slightly smaller than the brand text if the lines on the right are minimal. This was deemed a non-issue in real-world use, and this approach was applied as discussed.  You will find the discussion thread and video explaining this information [in Slack](https://sparkbox.slack.com/archives/CM9J5THPV/p1642528861056500).

With the border gone completely, it should be able to be zoomed at any amount without the pipes showing at different heights as one is gone.

## Need Review By (Date)
asap

## Urgency
medium

## Steps to Test
1. npm run test
2. Verify the right border on sulockup_cell1 is removed.
2. Verify on Chrome at zoom 67% no longer looks shifted and with two lines of different sizes.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
